playbooks/compute/nto/roles/configurecluster/tasks/detect_sno.yml

```diff
@@ -1,21 +1,18 @@
----
-- name: Retrieve worker nodes
+- name: Get Cluster version
```

Comment:
task name is wrong, you are getting the nodes info here not cluster version

Reply:
thanks - fixed